src/server/v2.0/handler/base.go

```diff
@@ -30,6 +31,5 @@ func (*BaseAPI) Prepare(ctx context.Context, operation string, params interface{
 
 // SendError returns response for the err
 func (*BaseAPI) SendError(ctx context.Context, err error) middleware.Responder {
-	// TODO: implement send error
-	return nil
+	return errs.NewErrResponder(err)
```

Comment:
Let's move this file out of v2.0 as it is shared by all versions of APIs

Reply:
Yes, it is shared by all versions of APIs (swagger generated APIs) so there is a `ErrResponder` in `server/error` pkg, `SendError` in `BaseAPI` is a shortcut.